rslabel_package/rslabel/ret.py

```diff
@@ -208,28 +260,35 @@ def app(args):
                 print('Deleting {}/'.format(directory))
                 shutil.rmtree(directory)
 
-        # Finally, upload the stats to the history folder on the
-        # server for stats tracking.
-        stats = [{'owner': data_owner,
-                  'stats': log,
-                  'date-time': datetime.datetime.now().isoformat()}]
-
-        # Only upload stats if someone has modified the annotations.
-        if log['images_labeled'] != 0 or log['images_validated'] != 0:
-            with tempfile.NamedTemporaryFile(prefix=data_owner) as tempf:
-                with open(tempf.name, 'w') as f:
-                    json.dump(stats, f)
-
-                with sftp.cd('history'):
-                    log_files = [x for x in sftp.listdir() if x.startswith(data_owner.replace(' ', '_'))]
-                    file_numbers = [int(os.path.splitext(x)[0].split('-')[-1]) for x in log_files]
-                    if len(file_numbers) == 0:
-                        f_number = 0
-                    else:
-                        f_number = max(file_numbers) + 1
-
-                    dst = os.path.basename(tempf.name)
-                    sftp.put(tempf.name)
-                    sftp.rename(dst, '{}-{}.log'.format(data_owner.replace(' ', '_'), f_number))
+            # Finally, upload the stats to the history folder on the
+            # server for stats tracking.
+        if tar_name not in clarification_tars:
```

Comment:
You need to update the stat tracking to include clarification jobs as well. Don't just switch around it like this. ;)

Reply:
I do not think that it is necessary, as clarification is for few people like me and Daylan. I do not need stats for it, if we want it I can add it in future. We need to push this ASAP, as our datasets are waiting on the sub to be uploaded for clarification which have lots of blurry images.